Cargo.toml

```diff
@@ -15,6 +15,12 @@ name = "rusoto"
 readme = "README.md"
 repository = "https://github.com/rusoto/rusoto"
 version = "0.24.0"
+homepage = "https://www.rusoto.org/"
+categories = [ "development-tools" ]
+
+[badges]
+travis-ci = { repository = "rusoto/rusoto" }
```

Comment:
Anyone have strong opinions on if we should specify the `master` branch here?  [Per the crates.io docs](http://doc.crates.io/manifest.html#package-metadata) `master` is the default but I generally prefer explicit to implicit.